cpp/src/arrow/adapters/orc/adapter.h

```diff
@@ -27,6 +27,7 @@
 #include "arrow/status.h"
 #include "arrow/type.h"
 #include "arrow/type_fwd.h"
+#include "arrow/util/key_value_metadata.h"
```

Comment:
No need for this a priori, as `KeyValueMetadata` is already declared in `arrow/type_fwd.h`.

Reply:
Ah I see..